.vimrc

```diff
@@ -84,8 +84,10 @@ set cursorline
 
 ""------------ Mappings --------------"
 "" Create tree navigation
-map <C-n> :vsp . <bar> vertical resize 30 <cr>
+map <C-n> :Lexplore <CR>
```

Comment:
I thought about another solution it should work whether you've got Lexplore
or not. So if your netrw version doesn't support Lexplore yet, it should
use Vexplore instead.

On Tue, Oct 20, 2015 at 11:52 AM Carlos Oliveira <carlospecter@gmail.com>
wrote:

> I rather keep things as they are. We can discuss this later, but if this
> is something that depends on a certain version of Vim that's not available
> to (almost) everyone, we should keep it quiet and rollback to the previous
> way to do things. Or change to Vexplore, and I think this is better that
> vsp . + vertical resize.
>
>
> On 10/20/2015 10:57 AM, Rodrigo Siqueira de Melo wrote:
>
> In .vimrc
> <https://github.com/rodrigosiqueira/lappis_vimrc/pull/11#discussion_r42491050>
> :
>
> > @@ -84,8 +84,10 @@ set cursorline
> >
> >  ""------------ Mappings --------------"
> >  "" Create tree navigation
> > -map <C-n> :vsp . <bar> vertical resize 30 <cr>
> > +map <C-n> :Lexplore <CR>
>
> Dude, how about keeps this part in the same way? Think about that, the old
> style works in every kind of vim. I believe this is a better option.
> However, if you disagree, how about to talk in google hangout or if it is
> possible talk personally?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rodrigosiqueira/lappis_vimrc/pull/11/files#r42491050>.
>
>
>